tests/sample_schemas/relative_ref/dereferenced_swagger.json

```diff
@@ -1,77 +1,98 @@
 {
-    "swagger": "2.0",
-    "info": {
-        "title": "Title was not specified",
-        "version": "0.1"
-    },
-    "produces": ["application/json"],
-    "paths": {
-        "/no_models": {
-            "get": {
-                "responses": {
-                    "200": {
-                        "description": "Return a standard_response",
-                        "schema": {
-                            "type": "object",
-                            "required": [
-                                "raw_response",
-                                "logging_info"
-                            ],
-                            "additionalProperties": false,
-                            "properties": {
-                                "raw_response": {
-                                    "type": "string"
-                                },
-                                "logging_info": {
-                                    "type": "object"
-                                }
-                            }
-                        }
-                    }
+  "swagger": "2.0",
```

Comment:
The file is not only re-indented, since its content is changed and adapted to the new output format produced by the ``resolve_references`` function.